kie-server-parent/kie-server-controller/kie-server-controller-impl/src/main/java/org/kie/server/controller/impl/storage/ControllerStorageFileWatcher.java

```diff
@@ -35,12 +30,16 @@
 public class ControllerStorageFileWatcher implements Runnable {
     
     private static final Logger log = LoggerFactory.getLogger(ControllerStorageFileWatcher.class);
+    public static final String STORAGE_FILE_WATCHER_INTERVAL = "org.kie.server.controller.templatefile.watcher.interval";
+    
+    private long sleepTime = Long.parseLong(System.getProperty(STORAGE_FILE_WATCHER_INTERVAL, "30000"));
```

Comment:
What if user makes a mistake and specifies value which is not valid long?
To make it more robuse we should probably try-catch parse exception, log warning and use default in such cases. Wdyt?

Reply:
this should fail fast as soon as user makes a mistake and give unexpected value and thus it will fail to boot